contracts/Rewards/RewardsDistributor.sol

```diff
@@ -445,10 +445,10 @@ contract RewardsDistributor is
     }
 
     /**
-     * @notice Set REWARD TOKEN last rewarding block for a single market.
-     * @param vToken market's whose reward token last rewarding block to be updated
-     * @param supplyLastRewardingBlockTimestamp New supply-side REWARD TOKEN last rewarding block for market
-     * @param borrowLastRewardingBlockTimestamp New borrow-side REWARD TOKEN last rewarding block for market
+     * @notice Set REWARD TOKEN last rewarding slot (block or second) for a single market.
+     * @param vToken market's whose reward token last rewarding slot (block or second) to be updated
+     * @param supplyLastRewardingBlockTimestamp New supply-side REWARD TOKEN last rewarding slot (block or second) for market
+     * @param borrowLastRewardingBlockTimestamp New borrow-side REWARD TOKEN last rewarding slot (block or second) for market
```

Comment:
It sets only and always the timestamp, not the block or the timestamp

Reply:
https://github.com/VenusProtocol/isolated-pools/pull/324/commits/25ed2b1ef2fc35b7f24628fdabadcd43d7037648